website_breadcrumb/views/templates.xml

```diff
@@ -6,6 +6,13 @@
 <data>
 
 <template id="breadcrumb" name="Breadcrumb snippet">
+    <t t-call="website_breadcrumb.breadcrumb-generator">
+        <t t-set='include_self' t-value='False'/>
+        <t t-set='revert_order' t-value='True'/>
+    </t>
+</template>
+
+<template id="breadcrumb-generator" name="Breadcrumb snippet generator">
```

Comment:
Adding a module that depends on another and disables what it does makes less sense than adding a module that adds something upon the previous one. That's why for me it seems more logical if the `website_snippet_breadcrumb` adds the template and logic (it should have the python part too), and then `website_breadcrumb` simply puts that on every page automatically.

If you still prefer to use manual mode, then please implement it in `website_breadcrumb` directly, to make usage consistent no matter if `website_snippet_breadcrumb` is installed. It should be enabled by default, since that's the original purpose of the module. Remember to update its README too.